data_acquisition/src/calc_center.R

```diff
@@ -0,0 +1,99 @@
+#' @title Calculate POI for polygons
+#' 
+#' @description
+#' Use polygon and 'point of inaccessibility' function (polylabelr::poi()) to 
+#' determine the equivalent of Chebyshev center, furthest point from every edge 
+#' of a polygon. POI function here will calculate distance in meters using the 
+#' UTM coordinate system and the POI location as Latitude/Longitude in WGS84 
+#' decimal degrees.
+#' 
+#' @param yaml contents of the yaml .csv file
```

Comment:
question: Should this say ".csv" or did you mean something else?

Reply:
Ohhh looks like maybe this was intentional, based on data_acquisition/src/format_yaml.R

Reply:
Totes! this is a weirdo, because of how I organized the yaml into multiple layers... though now that I think of it, I could probably use config::get() for each of the layers... but alas, maybe that's just aspirational and would add headaches....